eng/pipelines/templates/jobs/smoke.tests.yml

```diff
@@ -130,7 +130,6 @@ jobs:
         displayName: Install dev-tool
 
       - download: current
-        artifact: ${{parameters.ArtifactName}}
```

Comment:
This means it will download all the artifacts. Is that the intention?

Reply:
Yeah, because for the nightly run, there will be no ArtifactName being passed in and we'll have to grab the entire thing

Reply:
for the daily I assume we don't need this download at all so perhaps we should condition it out for those jobs instead?

Reply:
Not that we have the condition we probably should scope the the artifactName again.